modules/grizzly/src/main/java/org/glassfish/grizzly/attributes/AttributeBuilder.java

```diff
@@ -61,29 +63,18 @@ public interface AttributeBuilder {
      */
     <T> Attribute<T> createAttribute(String name, T defaultValue);
 
-    /**
-     * Create Attribute with name and initializer, which will be called, if Attribute's value is null on a AttributedObject
-     *
-     * @param <T> Type of attribute value
-     * @param name attribute name
-     * @param initializer NullaryFunction, which will be called, if Attribute's value is null on a AttributedObject
-     *
-     * @return Attribute<T>
-     */
-    <T> Attribute<T> createAttribute(String name, org.glassfish.grizzly.utils.NullaryFunction<T> initializer);
```

Comment:
With this removed, after switching client code to Supplier, the method left below is used, which is still marked as deprecated. This causes:
```
[WARNING] .../mq/main/mq-broker/broker-core/src/main/java/com/sun/messaging/jmq/jmsserver/service/imq/grizzly/GrizzlyMQPacketFilter.java:[42,13] <T>createAttribute(java.lang.String,org.glassfish.grizzly.attributes.NullaryFunction<T>) in org.glassfish.grizzly.attributes.AttributeBuilder has been deprecated
[INFO] 1 warning
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] .../mq/main/mq-broker/broker-core/src/main/java/com/sun/messaging/jmq/jmsserver/service/imq/grizzly/GrizzlyMQPacketFilter.java: warnings found and -Werror specified
[INFO] 1 error
```

Reply:
Looks like the `depreciation` left in place unintendedly